docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 # Emulator Manager
 
-You can use emulators to run your application in a virtual environment.
+After you have built your application, you need to test the application to ensure quality. To test and debug your application, you can configure a virtual environment before deploying your application. You can use emulators to run and test your application in a virtual environment. Tizen provides an emulator, Tizen Emulator Manager (Emulator Manager). It helps you to test your application in an environment similar to a real device.
 
-In order to test the application in a variety of environments, you need a variety of emulators. The Emulator Manager helps you easily create and manage emulator instances. Basically, the Emulator Manager allows you to generate emulator instances from a predefined platform and template. In addition, you can define the settings of the virtual device, such as skin, resolution, and hardware devices.
+Emulator Manager allows you to generate emulator instances from a predefined platform and template. Additionally, it helps you to define and configure settings such as skin, resolution, and hardware devices of your virtual device. 
 
-The main features of the Emulator Manager are:
+Emulator Manager helps you to:
```

Comment:
Tizen Docs doesn't use this documentation style.

We refer the guide, https://developers.google.com/style/lists#introductory-sentences-for-lists, and guide developers to follow the style.

Reply:
Done